.github/workflows/build.yml

```diff
@@ -20,10 +20,10 @@ jobs:
     # Install lint dependencies
     - uses: actions/setup-python@v2
       with:
-        python-version: '3.5'
+        python-version: '3.9'
```

Comment:
For windows, we don't have to change the minimum python and node version. (Also for local windows development, while installing the emsdk, it will install the required version of node and python package itself)
The problem is with the ubuntu build for that the python version has to updated to install properly (reverted node version changes)

Reply:
Thanks for the confirmation!